graphblas/io.py

```diff
@@ -482,6 +536,40 @@ def indices(self):
     return ret
 
 
+def to_pydata_sparse(A, format="coo"):
+    """Create a pydata.sparse array from a GraphBLAS Matrix or Vector
+
+    Parameters
+    ----------
+    A : Matrix or Vector
+        GraphBLAS object to be converted
+    format : str
+        {'coo', 'dok', 'gcxs'}
+
+    Returns
+    -------
+    sparse array (see https://sparse.pydata.org)
+
+    """
+    try:
+        from sparse import COO
+    except ImportError:  # pragma: no cover (import)
+        raise ImportError("sparse is required to export to pydata sparse") from None
+
+    format = format.lower()
+    if format not in {"coo", "dok", "gcxs"}:
+        raise ValueError(f"Invalid format: {format}")
+
+    # obtain an intermediate conversion via hardcoded 'coo' intermediate object
+    B = to_scipy_sparse(A, format="coo")
```

Comment:
If input is a `Matrix` and `format` is `"gcsx"`, maybe we should export to scipy `csr` array, which will be more efficient.